wheretofindme/views.py

```diff
@@ -49,6 +49,14 @@ def get_queryset(self):
         return Follow.objects.filter(from_user=self.request.user)
 
 
+class FollowersView(LoginRequiredMixin, ListView):
+    model = Follow
+    template_name = "wheretofindme/follower_list.html"
+
+    def get_queryset(self):
+        return Follow.objects.filter(to_user=self.request.user)
```

Comment:
We're gonna want an index on `to_user` to make this go reasonably fast on larger datasets.

Reply:
Looks like Django automatically adds indexes for ForeignKeys.